src/common.c

```diff
@@ -471,6 +471,20 @@ int ofi_addr_cmp(const struct fi_provider *prov, const struct sockaddr *sa1,
 	}
 }
 
+void ofi_straddr_log(const struct fi_provider *prov, enum fi_log_level level,
+		     enum fi_log_subsys subsys, char *log_str, const void *addr)
+{
+	char buf[OFI_ADDRSTRLEN];
+	uint32_t addr_format;
+	size_t len = sizeof(buf);
+
+	if (fi_log_enabled(prov, level, subsys)) {
+		addr_format = ofi_translate_addr_format(ofi_sa_family(addr));
+		fi_log(prov, level, subsys, __func__, __LINE__, "%s: %s\n",
```

Comment:
This will report ofi_straddr_log as the function with line number 483, which probably isn't what you want.  :/  I'm not sure what to do here.